### ISSUE TITLE
Issue in analysis in a browser using offline instrumentation

### ISSUE THREAD

**Issue by Zubairahmad09** [Samsung/jalangi2#165]
Hi:)

I want to take my analysis results using offline instrumentation in browser console (On Mac and Chrome browser). Here are the details

I used to run my javascript code file in html and then used the offline instrumentation on whole project by using the following command:

node src/js/commands/instrument.js --inlineIID --inlineSource -i --inlineJalangi --analysis src/js/commands/conditoncheck.js --outputDir /tmp test2/test3

The analysis works in terminal without showing any error. when i open the html file by giving the following command, It just gives the output of html file in console but not output of analysis on that file while same analysis works perfect in terminal.

file:///tmp/test2/test3/new.html

Don't know where I'm making mistake. Any solution in that case?

**Comment by msridhar** [comment]
> file:///tmp/test2/test3/new.html

That is not a command; you need to open that URL in a browser.  E.g., on Mac OS, I would do this to open an HTML file in Google Chrome:

```
open -a  /Applications/Google\ Chrome.app/ file.html
```

Does that help?

**Comment by Zubairahmad09** [comment]
Sorry for my typing mistake. The actual command is:

open file:///tmp/test3/new.html

Actually It opens the html file in Chrome browser but it only shows the program output in the console but not the analysis output.

**Comment by Zubairahmad09** [comment]
open -a  /Applications/Google\ Chrome.app/ file.html

Yes, the above commands also works but with same issue. just showing the output of program, not output of the analysis in the console.

**Comment by msridhar** [comment]
With a standard Jalangi analysis, you won't see the output of the analysis until you press Alt-Shift-T in the browser window.  This is documented under [Usage](https://github.com/Samsung/jalangi2#usage) under sub-heading "Analysis in a browser using offline instrumentation".  Does that solve the problem?

**Comment by Zubairahmad09** [comment]
So I took this simple analysis [](https://github.com/Samsung/jalangi2/blob/master/docs/tutorial1.md)

and when I run it in node.js using terminal by following command, it gives me analysis output as shown in above link. 

`node ../src/js/commands/jalangi.js --inlineIID --inlineSource --analysis analysis.js example.js`
 
But when I try to take the  analysis output in browser console by using offline instrumentation with following command

`node src/js/commands/instrument.js --inlineIID --inlineSource -i --inlineJalangi --analysis src/js/commands/analysis.js --outputDir /tmp test2/test3`

`open file:///tmp/test3/new.html
`
or
`open -a /Applications/Google\ Chrome.app/ new.html
`
and press Control-Cmd-I in case of mac, it just gives the output of the program in console not the analysis output like below.

Output:

foo
bar
foo
bar
foo
bar
foo
bar
foo
bar
done


Hope I explained it well. waiting for your answer. Thank you.
